java/ql/integration-tests/all-platforms/kotlin/logs/test.py

```diff
@@ -1,4 +1,39 @@
-from create_database_utils import *
+import json
+from math import log
+import pathlib
+import re
 
-run_codeql_database_create(['kotlinc test.kt'], test_db="kt-db", db=None, lang="java")
-run_codeql_database_create(['"%s" ./index_logs.py' % sys.executable], lang="java")
+
+def test(codeql, java_full, cwd: pathlib.Path, expected_files):
+    codeql.database.create(command=["kotlinc test.kt"])
+    expected_files.add("logs.actual")
+
+    with open("logs.actual", "w") as f_out:
+        log_dir = cwd / "test-db" / "log"
+        file_index = 0
+        for log_file in log_dir.glob("kotlin-extractor*.log"):
+            file_index += 1
```

Comment:
```suggestion
        for file_index, log_file in enumerate(log_dir.glob("kotlin-extractor*.log"), 1):
```

(but I really appreciate you did not port the existing script verbatim :heart: :laughing:)